scrapbook/utils.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+utils.py
+
+Provides the base API calls for scrapbook
```

Comment:
Hm. Is this the appropriate docstring for this?

Reply:
Opps, copy / paste error